src/popup/components/buttons/BtnBox.vue

```diff
@@ -1,98 +1,103 @@
 <template>
-  <Component
-    :is="component"
+  <BtnBase
+    v-bind="$attrs"
     class="btn-box"
-    :class="[
-      variant,
-      {
-        disabled,
-      },
-    ]"
-    :to="to"
-    :href="href"
-    :target="(href) ? '_blank' : null"
+    variant="muted"
+    :disabled="disabled"
+    :class="{
+      disabled,
+      'is-big': isBig,
+    }"
     v-on="$listeners"
   >
-    <slot />
-  </Component>
+    <IconWrapper
+      v-if="icon"
+      class="icon"
+      :icon="icon"
+      :is-boxed="isBig"
+    />
+
+    <span
+      v-if="text"
+      class="text"
+    >{{ text }}</span>
```

Comment:
```suggestion
    <span
      v-if="text"
      class="text"
    >
      {{ text }}
    </span>
```

Reply:
I did different approach with the `v-text` because I didn't want to introduce additional spaces to the rendered code.
This code:
```
<span>
  {{ text }}
</span>
```
is transpiled to:
```
<span> {{ text }} </span>
```
From my experience this can lead to some problems. Are you OK with this?

Reply:
Sure